README.md

```diff
@@ -10,47 +10,47 @@
     </a>
 </p>
 
-Red Hat Trusted Application Pipeline Installer (`rhtap-cli`)
+Red Hat Software Supply Chain installer (`ssc`)
 ------------------------------------------------------------
 
 # Abstract
 
-The `rhtap-cli` is designed as a sophisticated installer for Kubernetes [Helm Charts][helm], addressing the complexity of managing interdependent resources in Kubernetes environments. Unlike Kubernetes, which orchestrates resources individually without acknowledging their interdependencies, `rhtap-cli` enhances the deployment process by considering these relationships, thereby improving the user experience.
+The `ssc` binary is designed as a sophisticated installer for Kubernetes [Helm Charts][helm], addressing the complexity of managing interdependent resources in Kubernetes environments. Unlike Kubernetes, which orchestrates resources individually without acknowledging their interdependencies, `ssc` enhances the deployment process by considering these relationships, thereby improving the user experience.
```

Comment:
sophisticated installer **for** Kubernetes [Helm Charts][helm] ---> sophisticated installer **via** Kubernetes [Helm Charts][helm]

Reply:
Done